integration-cli/docker_cli_build_test.go

```diff
@@ -26,7 +26,6 @@ import (
 	"github.com/docker/docker/testutil/fakegit"
 	"github.com/docker/docker/testutil/fakestorage"
 	"github.com/moby/buildkit/frontend/dockerfile/command"
-	"github.com/opencontainers/go-digest"
```

Comment:
Hmm.. what happened here? Why was this import removed, or was it a duplicate import?

Reply:
Argh, my IDE messing things up. I thought I got them all, but clearly missed a couple.